qe_metrics/libs/orm_database.py

```diff
@@ -0,0 +1,166 @@
+from pony import orm
+from datetime import date, datetime
+
+from jira import Issue
+from simple_logger.logger import get_logger
+from typing import List, Any
+
+from pyaml_env import parse_config
+
+from qe_metrics.utils.general import verify_queries
+from qe_metrics.utils.general import format_issue_date
+
+DB_CONNECTION: orm.Database = orm.Database()
+
+
+class JiraIssuesEntity(DB_CONNECTION.Entity):
```

Comment:
JiraIsueEntity and ProductEntity should contain only DB data.
All other functions should be under Jira/Database classes.